packages/ra-ui-materialui/src/preferences/Configurable.tsx

```
@@ -73,6 +73,7 @@ export const Configurable = (props: ConfigurableProps) => {
     }
 
     const handleOpenEditor = () => {
+        if (!setEditor) return;
```

I think we should rather throw an error in this case